.github/workflows/ci.yml

```diff
@@ -46,8 +46,10 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v1
         with:
-          go-version: 1.14
+          go-version: 1.15
         id: go
 
       - name: build & test
-        run: cd airflow && go test
+        run: |
+            go test
+            cd airflow && go test
```

Comment:
Why we run tests two times?

Reply:
the first command runs the manually written e2e tests, the second command runs the generated tests from openapi-generator if there is any.

Reply:
What would you say to make a Makefile to run tests? It seems a common way, but no strong opinion

Reply:
Just a comment here : even being an old-timer and Bash-afficionado, I'd strongly discourage Makefiles to any new projects in favour of more modern tools.

Especially if you do not plan to use Makefile's build dependency management (which is the powerful part). In most part simple bash script is enough for simple cases, and in more complex ones https://bazel.build/ seems to be so much better choice and already pretty standard.

Also, the approach where you have separate commands/scripts in Ci depending on each other is good enough in most cases IMHO.

Reply:
i also don't have a strong opinion here. i usually use makefile for my own projects, but since Jarek has a strong opinion on this let's just not use it for now. I am also very hesitate to use heavy tools like bazel for simple tasks like this. i am going to leave it as is for now since it's just invoking simple `go test` command. But if any one of you have ideas on how we should further automate this, i am more than happy to update my PR.

Reply:
> the first command runs the manually written e2e tests, the second command runs the generated tests from openapi-generator if there is any.

Eventually, we may consider splitting those test into two steps to get better feedback, but that's a visual improvement

Reply:
good call, will make that change.